### ISSUE TITLE
[?] Weed Shop Enhancement Request

### ISSUE THREAD

**Issue by vicarsrd** [InZidiuZ/op-framework-issue-tracker#1749]
**Assurance:**
Have you read through the rules from the `README.md` file in the root folder of this GitHub repository?
Yes - o0ochanceo0o#5735

**Summary:**
Weed shop near MRPD was approved as an official business today.  Would like the following:
1. Keys for the building and all doors (including front entrance) DONE
2. Make storage / safe usable
3. Add Map location (Green marijuana leaf - Label: Green Wonderland)
5. Add edibles and blunts to the list that we could sell at Green Wonderland
- Make the edibles work like either the mechanic shop (allow us to use weed at the table in the back and turn it into edibles) or make it work like Bean Machine (put a spot in the back where we can purchase edibles).
- Requesting two types: Brownies (fill hunger, armor, and destress) and Gatorade (fill thirst, armor, and destress)
6. Make the plants in the back room automatically grow and harvest like the weed fields up north (but please yield more harvest for legal sales)
7. Replace the gumball machine with a working vending machine DONE
8. Add a computer in the back where we could purchase supplies at a lower price (50%) of the super store (buckets, fertilizer, seeds)
9. Request job titles (Salesperson 1, Salesperson 2, Salesperson 3, Manager, Owner) and applicable salaries.
10. Request t-shirts with Green Wonderland logo on front left breast (small) and large logo on back.  Request one white and one black.  DONE

**Reason:**
The business is approved and I received the license today.  Would like the shop functional for myself and my employees.  Also need ability to give keys to employees and remove from fired employees (as needed).

**Comment by vicarsrd** [comment]
![image](https://user-images.githubusercontent.com/81762234/114601873-3b893b00-9c53-11eb-87b5-47116fa4c85b.png)

**Comment by vicarsrd** [comment]
That's the sign we would like out front of our business to replace Best Buds

**Comment by vicarsrd** [comment]
![image](https://user-images.githubusercontent.com/81762234/114608331-d0dbfd80-9c5a-11eb-8a46-d7892f288b79.png)

**Comment by LikeConnorr** [comment]
+1 on behalf of the medical department supporting this business. Excellent proposals & delivery of a company with high potential.

**Comment by vicarsrd** [comment]
Also would like to add a map to this location!  And thank you for the kind words @LikeConnorr !!!

**Comment by Djmeowzle** [comment]
-1 Stop posting advertisements for the GitHub in general chat

**Comment by riedling11** [comment]
+1 this will be very valuable to the community!

**Comment by vicarsrd** [comment]
^^^Is there any movement on this or help needed?

**Comment by adarchimedes** [comment]
I'm manager now at Green Wonderland, and I am inquiring about how to get keys available at the building, and eventually safe storage. Archimedes#4974 thank you in advance!

**Comment by vicarsrd** [comment]
We have been doing a lot more business since opening to the public (the city and justice dept granted us the ability to sell recreationally).  We are asking for the above fixes, if available.  Any help or update would be greatly appreciated!

**Comment by vicarsrd** [comment]
Deleted request to have a spot for purchase while employees are away.  Due to legal reasons in the city, we cannot have this: we keep a log of all sales and people are limited to how much they can purchase at a time recreationally (5/hr) and medicinally (10/hr).

**Comment by antihero6713** [comment]
1 and 7 completed
Added locks to  doors

Added snack vending machine, and moved watercooler

**Comment by vicarsrd** [comment]
This is in the Priority Board now; do we have an update on the status of this?

**Comment by antihero6713** [comment]
I know door locks will be on the building. Not sure if the other things.  Still have to wait on the editable models

**Comment by vicarsrd** [comment]
@antihero6713 thank you sir!!!

**Comment by vicarsrd** [comment]
Can use 140 radar_weed_stash from this list in green for the map blip: https://docs.fivem.net/docs/game-references/blips/

**Comment by vicarsrd** [comment]
... or 496 radar_production_weed in green (if possible)

**Comment by vicarsrd** [comment]
Do we have movement on this card?  Would it be more beneficial for me to break this out into separate cards or is everything okay here?

**Comment by tftwsaaron** [comment]
any word on jobs and keys.

**Comment by vicarsrd** [comment]
Thank you for the t-shirts!!! We love them!  Any movement on the rest of the card?

**Comment by vicarsrd** [comment]
Hi Team, Thanks so much for your hard work so far.  Is there anything we can do to start some movement on this card?

**Comment by DaxterPancake** [comment]
Closing in favour of #1983 #1982 #1981
